lib/flame/dispatcher.rb

```diff
@@ -86,6 +86,13 @@ def config
 			@app_class.config
 		end
 
+		## Available routes endpoint
+		def available_endpoint
+			return @available_endpoint if defined? @available_endpoint
+			@available_endpoint ||=
```

Comment:
There is no need to use `||` if there is `defined?`

Reply:
yeah. changed.